Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ClassicMenu/ClassicMenuPage.xaml

```diff
@@ -0,0 +1,270 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.ClassicMenuPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:commands="using:Microsoft.Toolkit.Uwp.SampleApp.ClassicMenu.Commands"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <commands:NewProjectCommand x:Key="NewProject" />
+        <commands:NewFileCommand x:Key="NewFile" />
+        <commands:GenericCommand x:Key="GenericCommand" />
+        <Style TargetType="controls:ClassicMenuItem">
+            <Setter Property="Background" Value="{StaticResource Brush-Grey-01}" />
+            <Setter Property="Foreground" Value="White" />
+        </Style>
+
+        <Style TargetType="MenuFlyoutSeparator">
+            <Setter Property="Background" Value="White" />
+        </Style>
+
+        <Style x:Key="MenuFlyoutItemStyle"
```

Comment:
For a reason or another the style is not applied for the second level menu items !!